anaconda_project/archiver.py

```diff
@@ -148,6 +148,9 @@ def _load_ignore_file(project_directory, frontend):
 
 def _git_ignored_files(project_directory, frontend):
     if not os.path.exists(os.path.join(project_directory, ".git")):
+        if os.path.exists(os.path.join(project_directory, ".gitignore")):
```

Comment:
Offers a warning message if this is not a git repo but .gitignore is found. In the future we should parse this file internally instead of relying on git